src/server/game/Entities/AreaTrigger/AreaTrigger.cpp

```diff
@@ -187,7 +197,17 @@ void AreaTrigger::SetDuration(int32 newDuration)
 {
     _duration = newDuration;
     _totalDuration = newDuration;
-    SetUInt32Value(AREATRIGGER_DURATION, newDuration);
+
+    // negative duration (permanent areatrigger) sent as 0
+    SetUInt32Value(AREATRIGGER_DURATION, std::max(newDuration, 0));
+}
+
+void AreaTrigger::_UpdateDuration(int32 newDuration)
+{
+    _duration = newDuration;
+
+    // don't broadcast update
+    UpdateUInt32Value(AREATRIGGER_DURATION, _duration);
```

Comment:
but UpdateUInt32Value sends the value (always, it skips the check if new value is different to old one)